assets/templates/geography-start.tmpl

```diff
@@ -0,0 +1,43 @@
+{{ $lang := .Language }}
+
+<div class="ons-page__container ons-container ons-u-mb-xl">
+    <div class="ons-grid ons-js-toc-container ons-u-ml-no">
```

Comment:
Your page has an additional `padding-left` that has skewed the vertical alignment of the page. If you remove `ons-u-ml-no` this resolved the problem
```suggestion
    <div class="ons-grid ons-js-toc-container">
```

Reply:
Removed the no margin class